report_thesis/src/sections/introduction.tex

```diff
@@ -5,30 +5,42 @@ \section{Introduction}\label{sec:introduction}
-These effects can cause the intensity of emission lines from an element to vary independently of that element's concentration, introducing unknown variables that complicate the analysis. 
-Furthermore, due to the high cost of data collection, datasets are often small, which further complicates the task of building accurate and robust models.
+However, predicting major oxide compositions from \gls{libs} data still presents significant computational challenges.
+These include the high dimensionality and non-linearity of the data, compounded by issues of multicollinearity and matrix effects~\cite{andersonImprovedAccuracyQuantitative2017}.
+Such effects can cause the intensity of emission lines from an element to vary independently of that element's concentration, introducing unknown variables that complicate the analysis.
+Furthermore, the high cost of data collection often results in small datasets, exacerbating the difficulty of building accurate and robust models.
 
-Various machine learning models have been used to predict the composition of major oxides in the sample, including \glspl{cnn}~\cite{yang_laser-induced_2022, yangConvolutionalNeuralNetwork2022}, \gls{svr}~\cite{rezaei_dimensionality_reduction}, and hybrid models like \gls{df}-\gls{k-elm}~\cite{song_DF-K-ELM} that incorporate domain knowledge to enhance model interpretability and performance.
-However, the high dimensionality and multicollinearity of the spectral data remains a significant challenge for these models.
+Previous work has aimed to improve the prediction of major oxide compositions from LIBS data.
+Regression techniques and dimensionality reduction with feature selection have been used to enhance the accuracy and interpretability of these models.
+Tailored approaches have also been developed, where different models are selected based on their performance with specific spectral characteristics and distances~cite{rezaei_dimensionality_reduction, andersonPostlandingMajorElement2022}.
+Moreover, models incorporating physical principles have demonstrated improved accuracy by handling residuals that traditional models fail to explain~\cite{song_DF-K-ELM}.
+However, predicting oxide compositions remains challenging due to the complex, nonlinear nature of LIBS data.
+This underscores the need for continued research into more adaptive and robust machine learning strategies to tackle these issues effectively.
 
-Building upon the baseline established in~\citet{p9_paper}, this thesis aims to explore approaches for tackling the challenges in predicting major oxide compositions from \gls{libs} data. We develop machine learning models that seek to enhance the accuracy and robustness of these predictions.
+This thesis aims to improve the prediction of major oxide compositions from LIBS data by building on the foundation established in~\citet{p9_paper} in order to enhance model accuracy and robustness.
```

Comment:
`\gls{libs}`

There's something about this sentence structure that isn't right to me.

We aim to improve predictions of elemental concentrations from LIBS data → by building on our previous work → to improve the model accuracy and robustness.

I'd rather focus on the goals in one sentence, and then mention what we're building on in another.
Get to the point when stating a goal.

Like:

> This thesis aims to improve the model accuracy and robustness in predicting major oxide compositions from \gls{libs} data.

And then say something like `we build on the foundation laid in ...`.

Perhaps we should also include the part about 'tackling the challenges in predicting... from LIBS data?'

Reply:
Rephrased this